pvactools/tools/pvacseq/generate_protein_fasta.py

```diff
@@ -193,6 +193,12 @@ def main(args_input = sys.argv[1:]):
 
     temp_dir = tempfile.mkdtemp()
     proximal_variants_tsv = convert_vcf(args.input_vcf, temp_dir, args.sample_name, args.phased_proximal_variants_vcf, args.flanking_sequence_length, args.pass_only)
+    if proximal_variants_tsv is not None:
+        proximal_variants_file = "{}.proximal_variants.tsv".format(args.output_file)
+        with open(proximal_variants_tsv, 'r') as input_file:
+            with open(proximal_variants_file, 'w', newline='') as output_file:
+                output_file.write(input_file.read())
```

Comment:
While this works just fine, the more pytonic way to do this would be to use the shutil package. `shutil.copy(source_file_path, destination_file_path)` will copy a file from one location to another.

Reply:
I changed to this command and tested! Works great!